test/3x/torch/quantization/test_static_quant.py

```diff
@@ -49,6 +49,21 @@ def test_static_quant_default(self):
         q_model = quantize(fp32_model, quant_config=quant_config, run_fn=run_fn, example_inputs=example_inputs)
         assert q_model is not None, "Quantization failed!"
 
+    @pytest.mark.skipif(not is_ipex_available(), reason="Requires IPEX")
+    def test_static_quant_fallback(self):
+        fp32_model = copy.deepcopy(self.fp32_model)
+        quant_config = get_default_static_config()
+        example_inputs = self.input
+        # fallback by op_type
+        quant_config.set_local(torch.nn.modules.linear.Linear, StaticQuantConfig(w_dtype="fp32", act_dtype="fp32"))

```

Comment:
Please use string 'Linear' instead of objective. Objective is more complex and cannot be saved directly as json.

Reply:
Aligned with @yiliu30, we will accept original torch class and method as input of `set_local` to avoid confusion with op_name and use its string name internally to map with IPEX op_type.